Components/WdBlockDriver/WdBlockDriver.hpp

```diff
@@ -0,0 +1,82 @@
+// ======================================================================
+// \title  WdBlockDriver.hpp
+// \author ortega
+// \brief  hpp file for WdBlockDriver component implementation class
+// ======================================================================
+
+#ifndef Components_WdBlockDriver_HPP
+#define Components_WdBlockDriver_HPP
+
+#include <wdLib.h>
+#include "Components/WdBlockDriver/WdBlockDriverComponentAc.hpp"
+
+namespace Components {
+
+class WdBlockDriver : public WdBlockDriverComponentBase {
+  public:
+    // ----------------------------------------------------------------------
+    // Component construction and destruction
+    // ----------------------------------------------------------------------
+
+    //! Construct WdBlockDriver object
+    WdBlockDriver(const char* const compName  //!< The component name
+    );
+
+    //! Destroy WdBlockDriver object
+    ~WdBlockDriver();
+
+    void callIsr(U32 ticks);
+
```

Comment:
Add a public function `stopTimer()`. It can call `wdCancel()` and the ISR will no longer be invoked.